ext-apiserver/pkg/controller/machine/controller_test.go

```diff
@@ -48,26 +47,31 @@ var _ = Describe("Machine controller", func() {
 
 	Describe("when creating a new object", func() {
 		It("invoke the reconcile method", func() {
+			cluster := Cluster{}
+			cluster.Name = "cluster-1"
+			_, err := cs.ClusterV1alpha1().Clusters("machine-controller-test-handler").Create(&cluster)
+			Expect(err).ShouldNot(HaveOccurred())
 			client = cs.ClusterV1alpha1().Machines("machine-controller-test-handler")
```

Comment:
I'm surprised that it's working with a different namespace. The current code is assuming the default namespace is many locations.

Reply:
What code? The actuator? I stubbed that out with the logging one for tests. The general informer framework shouldn't have namespace issues.

Reply:
The controller code has some namespace logic there. I suspect the test will fail with my #572 PR. Maybe just start the test with default namespace. We can change that once we have different namespace support.

Reply:
Done